modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/GridCacheQueryManager.java

```diff
@@ -1143,10 +1159,9 @@ protected void runQuery(GridCacheQueryInfo qryInfo) {
                 IgniteSpiCloseableIterator iter;
                 GridCacheQueryType type;
 
-                res = loc ?
-                    executeQuery(qry, trans, loc, taskName,
-                        recipient(qryInfo.senderId(), qryInfo.requestId())) :
-                    queryResult(qryInfo, taskName);
+                res = loc
```

Comment:
Original code formating (`?` at the EOL) is more common in Ignite.

Reply:
Both style are acceptable, as we discussed privately let's keep it as is.